arch/xtensa/Kconfig

```diff
@@ -90,4 +90,13 @@ config XTENSA_WAITI_BUG
 	  platforms which prefixes a WAITI entry with 128 NOP
 	  instructions followed by an ISYNC and EXTW.
 
+config XTENSA_SMALL_VECTOR_TABLE_ENTRY
+	bool "Enable workaround for small vector table entries"
+	default n
```

Comment:
You can remove this line, bools are implicitly default n.

Reply:
Done!

Reply:
Thank you!